### PR TITLE
Join Our Community! page revamp

### LINKED ISSUE #1068: Join Our Community! page revamp
Figma design:
https://www.figma.com/file/UAuzPXadk1D5Wem0c8nXwq/Marketing-Site-Copy?node-id=0%3A1

Assigned to: @lukas-kasticky

### PR DESCRIPTION
**Pull request by lukas-kasticky**
https://github.com/thenewboston-developers/Design/issues/22

closes #1068 

I created `MarketingCard` from `MarketingButton`, because the former was used in other places and using the social logo marks wouldn't have worked.

### CODE REVIEW

**Comment by lukas-kasticky** [comment]
maybe @kristykjlee wants to check this out before merging?

**Comment by kristykjlee** [comment]
> maybe @kristykjlee wants to check this out before merging?

Design looks good! Ready to merge

File: src/utils/social-media.ts [hunk 1]
```diff
@@ -9,3 +9,27 @@ export const socialMediaUrls = {
...
+
+export const socialMediaDescriptions = {
+  [SocialMedia.facebook]: 'Stay up to date by following us on our Facebook page.',
+  [SocialMedia.github]:
+    'You can find the official repositories for our projects on our GitHub and start contributing today.',
+  [SocialMedia.linkedin]: 'Follow us on LinkedIn and expand your network.',
+  [SocialMedia.reddit]:
+    'Our Reddit community is built to help others learn about computer science and other academic subjects.',
+  [SocialMedia.slack]:
+    'With 2049 members our Slack team is the central communication channel for app ideas, development & design.',
```
Comment (jamessspanggg) [inline src/utils/social-media.ts:31]: Would it be more appropriate to be `With over thousands of members...` as the member count is dynamic?

**Review by jamessspanggg — APPROVED** [review]

**Comment by thesanjeevsharma** [comment]
@lukas-kasticky @kristykjlee 
image
1. How about we reduce the padding on sides on mobile screens? Right now it's at 48px, I think 28px would be fine.
2. Logo sizes are at 64px. Should we do 48px on mobile screens? 
3. Everything is right aligned. Should we center align content on mobile screen?

image
Just confirming, if we want these to be flowing as 1 column, or we can do 2 column grid for it.

Once you confirm, I'll merge this.

**Comment by lukas-kasticky** [comment]
i reduced the padding to `32px`, just like on the index page. i also made the logos smaller and `left`-aligned everything, `center` doesn't look good imo.

i think it gets a little bit too messy when it's in two columns.

Screenshot_2020-11-28 Social thenewboston

**Comment by kristykjlee** [comment]
@thesanjeevsharma Here is a link to the design you asked for
https://www.figma.com/file/UAuzPXadk1D5Wem0c8nXwq/Marketing-Site-Copy?node-id=0%3A1

**Comment by lukas-kasticky** [comment]
> https://www.figma.com/file/UAuzPXadk1D5Wem0c8nXwq/Marketing-Site-Copy?node-id=0%3A1

should i update the changes in the figma projects?

**Review by thesanjeevsharma — APPROVED** [review]

**Pull request merged**